content/technotes/2018-08-08-phylotar-release.md

```diff
@@ -0,0 +1,173 @@
+# run the pipeline
+run(wd = wd)
+# ^^ running should take about 5 minutes to complete
+```
+
+For more details on running the pipeline, such as changing the parameters or understanding the results, see the vignette, `vignette('phylotaR')` or visit the [website](https://ropensci.github.io/phylotaR/).
+
+### Timings
+
+The time it takes for a pipeline to complete depends on the number of taxonomic groups a taxon contains and the number of sequences it represents. The genus Aotus has more species than the great apes but the latter will take far long because it represents highly well-sequenced species.
+
+Taxon|N. taxa|N. sequences|Time taken (mins.)|
+|:--|--:|--:|--:|
+Anisoptera|1175|11432|72|
+Acipenseridae|51|2407|13|
+Tinamiformes|25|251|2.7|
+Aotus|13|1499|3.9|
+Bromeliaceae|1171|9833|66|
+Cycadidae|353|8331|37|
+Eutardigrada|261|960|14|
+Kazachstania|40|623|23|
+Platyrrhini|212|12731|60|
+
+### Reviewing the results
+
+After a pipeline has completed, the `wd` will contain all the results files. The information contained within these files can be read into the R environment using the `read_phylota()` function. It will generate a `Phylota` object that contains information on all the identified 'clusters' -- i.e. groups of ortholgous sequences. A user can interact with the object to filter out unwanted sequences, clusters or taxonomic groups. The package comes with completed results for playing with. In the example below, we demonstrate how to generate a presence/absence matrix of all the genera in the top 10 clusters identifed for cycads.
+
+```r
+library(phylotaR)
+data(cycads)
+# drop all but first ten
+cycads <- drop_clstrs(cycads, cycads@cids[1:10])
+# get genus-level taxonomic names
+genus_txids <- get_txids(cycads, txids = cycads@txids, rnk = 'genus')
+genus_txids <- unique(genus_txids)
+# dropping missing
+genus_txids <- genus_txids[genus_txids !=  '']
+genus_nms <- get_tx_slot(cycads, genus_txids, slt_nm = 'scnm')
+# make alphabetical for plotting
+genus_nms <- sort(genus_nms, decreasing = TRUE)
+# generate geom_object
+p <- plot_phylota_pa(phylota = cycads, cids = cycads@cids, txids = genus_txids,
+                     txnms = genus_nms)
+# plot
+print(p)
+```
+
+![presenceabsence of cycad genera](/img/blog-images/2018-08-08-phylotar/pa.png)
+
+In this next example, we create a treemap showing the differences in the number of sequences and clusters identifed between genera in tinamous.
+
+```r
+library(phylotaR)
+data(tinamous)
+# Plot taxa, size by n. sq, fill by ncl
+txids <- get_txids(tinamous, txids = tinamous@txids, rnk = 'genus')
+txids <- txids[txids !=  '']
+txids <- unique(txids)
+txnms <- get_tx_slot(tinamous, txids, slt_nm = 'scnm')
+p <- plot_phylota_treemap(phylota = tinamous, txids = txids, txnms = txnms,
+                          area = 'nsq', fill = 'ncl')
+print(p)
+```
+
+![treemap of tinmous genera](/img/blog-images/2018-08-08-phylotar/treemap.png)
```

Comment:
beneath the treemap can you give a brief text explanation of what it shows re: "the differences in the number of sequences and clusters identifed between genera in tinamous"?